portfolio/src/main/webapp/index.html

```diff
@@ -2,20 +2,18 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="indexstyle.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>Caroline Hana</h1>
-      <p>This is my portfolio.</p>
-      <p> Im an 18 year old college student from New York City!</p>
-     <img src="/Images/profile2.jpg" alt="Caroline" width="300" height="400">
-     <img src="/Images/avatar.jpg" alt="Caroline" width="300" height="300">
-      <button onclick="addRandomFunFacts()">Press Me!</button>
-      
-      <div id="facts-container"></div>
+    <div class="DefaultPageImage"></div>
+    <div class ="MainPageText">
+      <h1>Welcome to my blog!</h1>
+      <h2> Select your preferred language below: </h2>
+      <button onclick="EnglishPage()">English</button>
```

Comment:
You could also try <a href="/main_ENG.html"> English </a>. You won't need any JavaScript in that case.

Reply:
I agree. The code is much clearer and simple by removing the javascript function.